shared/src/convention/conventionRefinements.ts

```diff
@@ -53,6 +54,26 @@ export const isTutorEmailDifferentThanBeneficiaryRelatedEmails = (
   return !emails.includes(establishmentTutor.email);
 };
 
+export const minorBeneficiaryHasRepresentative = ({
+  dateStart,
+  signatories,
+  dateSubmission,
+}: ConventionDto) => {
+  const beneficiaryAgeAtConventionStart = differenceInYears(
+    new Date(dateStart),
+    new Date(signatories.beneficiary.birthdate),
+  );
+
+  const ruleAppliesFrom = new Date("2023-10-28");
+  const conventionWasSubmittedBeforeRuleApplies =
+    new Date(dateSubmission).getTime() < ruleAppliesFrom.getTime();
+
+  if (conventionWasSubmittedBeforeRuleApplies) return true;
+  if (beneficiaryAgeAtConventionStart >= 18) return true;
+
+  return !!signatories.beneficiaryRepresentative;
```

Comment:
```suggestion
  return conventionWasSubmittedBeforeRuleApplies ||  beneficiaryAgeAtConventionStart >= 18 || !!signatories.beneficiaryRepresentative;
```
Ça ferait pas le boulot ?

Reply:
ca ferait, je prends